src/components/popup/balances-and-activity.tsx

```diff
@@ -27,16 +32,44 @@ function EmptyActivity() {
   );
 }
 
+const LocalTxListItem = ({ txid }: { txid: string }) => {
+  const localTx = useCurrentAccountLocalStacksTransaction(txid);
+  if (localTx) return <LocalTxItem txid={txid} transaction={localTx.transaction} />;
+  return <>Loading...</>;
+};
+
+const LocalTxList = ({ txids }: { txids: string[] }) => {
+  return (
+    <Stack pb="extra-loose" spacing="extra-loose">
+      <Box>
+        <Text textStyle="body.small" color={color('text-caption')}>
+          Submitted
+        </Text>
+        <Stack mt="base-loose" spacing="loose">
+          {txids.map(txid => (
+            <LocalTxListItem txid={txid} key={txid} />
```

Comment:
Should we limit the amount of submitted transactions we show? If the list item doesn't link to the explorer (which it can't), what does this really do a for a user? If submitted transactions build up, would just showing a number of tx submitted be enough feedback? `You have 5 submitted transactions...` or some feedback like that?

Reply:
We can link to the explorer, there is a valid txid and it will ultimately resolve to the transaction when it's available

Reply:
What information is it showing at that point if the tx isn't `pending` yet?

Reply:
While true, I can see the multiples of people on Discord asking where the tx is (at least when the network is as it is). I would avoid this.

I would show all the pending txs, but not link to the explorer (a dead end when pending). If there are loads of submitted txs, there's a bigger problem at hand.

Reply:
My thought is, if we do this, those shouldn't have a hover/link and should be designed differently to let users know there isn't much information we can provide ...so there is no false hope opening a link to the explorer.

Reply:
sure, if there's no action, we should def also remove the hover actions too `usePressable(false)`

Reply:
yeah, i mean at the bottom line though these are valid transactions. they will eventually confirmed. so there is no false hope or anything. but I agree, i think having them be of a different style, eg greyed out or a different status, and not clickable makes sense.

Having it clickable does give the user the ability to get the txid, which if there are issues, we should provide a way for them to get the txid.

Reply:
Is there anything we can do in the explorer to provide better feedback? Prob not in this state?

Reply:
Interesting thought. Probably overcomplicating it, but if it had a `?submitted` param we could totally show a different state there, too.